docs/APIs.md

```diff
@@ -1,23 +1,43 @@
 <p>
+
```

Comment:
Not sure why all the new lines were added. Possibly a different version of swagger. This change used v0.31.0.